test/unit/input_convertors.py

```diff
@@ -4,7 +4,7 @@
 import numpy as np
```

Comment:
nit: should the file name be "converters" with E instead of O?

Reply:
That's a question for another time.